### PR TITLE
Update

### PR DESCRIPTION
**Pull request by btkostner**
This rewrites this to be a single nuxt app for zendesk.

TODO:
- Update README
- CI (possibly another PR instead)

The way I've been testing is `npm run build` and then `zat server` inside the `dist` folder

### CODE REVIEW

**Review by doomspork — APPROVED** [review]
:tada:

**Comment by doomspork** [comment]
Let's get this deployed into staging and production before going down the rabbithole of CI/CD.

**Review by doomspork — APPROVED** [review]

File: pages/support/ticket-sidebar.vue [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    <div
+      v-if="user.id"
+      class="u-mb"
+    >
+      <h2 class="u-fs-lg u-mb-sm">User {{ user.id }}</h2>
+
+      <div>
+        <a
+          :href="`https://admin.system76.com/account/customers/${user.id}`"
```
Comment (doomspork) [inline pages/support/ticket-sidebar.vue]: We should probably make the host configurable. We went to a lot of trouble to not configure field names but now there is a required code change per environment.

We should refactor this stuff to take advantage of configuration and simpler code.

Comment (btkostner) [reply pages/support/ticket-sidebar.vue]: I second this. I'll update / simplify this in another PR

**Pull request merged**